### PR TITLE
React v18 Compatibility: Install Framer Motion and Add Flip Words Animation to Auth Pages

### PR DESCRIPTION
**Pull request by JoseChaconUrias**
TL;DR: Change React version from v19 to v18; Installed and added Aceternity Flip Words animation to authentication pages.

---

***IMPORTANT***

Changing React version from v19 to v18! This is because Framer Motion requires v18 of React. Framer Motion is an open source library that is still not compatible with the newest version of React.

The current version of Next.js is v15 which is compatible with React v18, so no major issues should arise from using React 18 with this Next.js version.

---

### New Dependencies Added
- `framer-motion@11.11.11`
- `clsx@2.1.1`
- `tailwind-merge@2.5.4`

### Reason for Adding Dependencies
- `framer-motion`: To handle animations in our components.
- `clsx`: For easier conditional class management.
- `tailwind-merge`: To help merge and deduplicate Tailwind classes.

### Installation Instructions
After pulling this branch, please run:
```bash
npm install
```
---

### Why Aceternity
Aceternity UI provides ready-to-use, customizable components like buttons, modals, forms, and notifications. This can save you a lot of time compared to building these components from scratch, especially for common elements found in most applications. Aceternity UI typically allows for easy customization and theming, enabling you to adjust styles to match your brand or specific project requirements. This includes options for modifying colors, typography, and layouts.

### Changes made
In this pull request I am utilizing Flip Words. Adding an attention grabbing description of the site while keeping the layout simplistic. This gives the user a quick introduction before registering for the app.

https://github.com/user-attachments/assets/fa00606a-ef93-454c-997b-5c140ece2c2e

### Key information
This utility function cn combines clsx and tailwind-merge to streamline the handling of dynamic and conditional Tailwind CSS classes.

This can be found in /src/lib/utils.ts
```typescript
import { ClassValue, clsx } from "clsx";
import { twMerge } from "tailwind-merge";

export function cn(...inputs: ClassValue[]) {
  return twMerge(clsx(inputs));
}
```

For more information visit: https://ui.aceternity.com/components/flip-words

### CODE REVIEW

**Review by JoseChaconUrias — COMMENTED** [review]
Correct
